### DISCUSSION TITLE
第4节课作业(9班)

### DISCUSSION THREAD

**Discussion by SaigeRing** [InternLM/Tutorial#257] [作业]
**基础作业：**

构建数据集，使用 XTuner 微调 InternLM-Chat-7B 模型, 让模型学习到它是你的智能小助手，效果如下图所示，本作业训练出来的模型的输出需要将 **不要葱姜蒜大佬** 替换成 自己名字或昵称！

**微调前**（回答比较官方）
![image](https://github.com/InternLM/tutorial/assets/154900748/72c0364a-4a03-4d91-8f5e-7cf0b73a27a2)

**微调后**（对自己的身份有了清晰的认知）
![image](https://github.com/InternLM/tutorial/assets/154900748/b55d63ec-c634-414a-b5ae-8a5a23254f56)

作业参考答案：https://github.com/InternLM/tutorial/blob/main/xtuner/self.md

**进阶作业：**

- 将训练好的Adapter模型权重上传到 OpenXLab、Hugging Face 或者 ModelScope 任一一平台。
- 将训练好后的模型应用部署到 OpenXLab 平台，参考部署文档请访问：https://aicarrier.feishu.cn/docx/MQH6dygcKolG37x0ekcc4oZhnCe

**Comment by liuxianyi** [comment]
[📒标记的为作业内容](https://liuxianyi.notion.site/XTuner-800a548e6822495e848d7c7e3041c747?pvs=4)

**Comment by xiaomile** [comment]
[第四课作业](https://github.com/xiaomile/InternLM-homework/tree/main/%E4%BD%9C%E4%B8%9A3)

**Comment by SaigeRing** [comment]
https://zhuanlan.zhihu.com/p/677700606

**Comment by cookieswolf** [comment]
[04-XTuner 大模型单卡低成本微调实战](https://www.mubu.com/doc/5sUvV-0TB3O)

**Comment by ShelterWFF** [comment]
class4: http://t.csdnimg.cn/o4J7m

**Comment by li4692625** [comment]
https://blog.csdn.net/li4692625/article/details/135577097

**Comment by nxznm** [comment]
https://zhuanlan.zhihu.com/p/677742755

**Comment by KJ-Huang-sz** [comment]
https://blog.csdn.net/qq_40831032/article/details/135575616

**Comment by nini0919** [comment]
https://morning-gaura-eed.notion.site/44511e75ab17414bb438c1e348d97cb5?pvs=4

**Comment by RangeKing** [comment]
https://rangeking.github.io/#/page/4.%20xtuner%20hands-on

**Comment by AIzealotwu** [comment]
https://blog.csdn.net/m0_72806612/article/details/135584305?spm=1001.2014.3001.5502

**Comment by haizeigh** [comment]
https://note.youdao.com/s/6Y62k3R4

**Comment by wangxuejiao** [comment]
https://zhuanlan.zhihu.com/p/677763102
